iceoryx2-cal/src/communication_channel/message_queue.rs

```diff
@@ -138,23 +138,25 @@ impl<T: Copy + Debug> CommunicationChannel<T> for Channel<T> {
 }
 
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
-pub struct Configuration {
+pub struct Configuration<T: Copy + Debug> {
```

Comment:
Out of curiosity, why was this changed required?

If it is just to be able to call `Channel::<T>::default_*()` it would also work with `Channel::<()>::default_*()`.

Or alternatively just replacing the constants with free functions would also work without having to add generic parameter all over the place.

Reply:
There was actually no need for it and I removed it again.

Reply:
Weird, I still see the generic parameter in the diff

Reply:
Whoopsie my mistake. I need more sleep :/